documentation/modules/configuring/proc-config-mirrormaker2-replication.adoc

```diff
@@ -111,38 +111,44 @@ spec:
         replication.factor: 1 <20>
         offset-syncs.topic.replication.factor: 1 <21>
         sync.topic.acls.enabled: "false" <22>
-        replication.policy.separator: "" <23>
-        replication.policy.class: "io.strimzi.kafka.connect.mirror.IdentityReplicationPolicy" <24>
-    heartbeatConnector: <25>
+        refresh.topics.interval.seconds: 60 <23>
+        refresh.groups.interval.seconds: 600 <24>
```

Comment:
I think this needs to be in the `checkpointConnector` section.

Reply:
Moved